src/seat.c

```diff
@@ -32,17 +32,28 @@ input_device_destroy(struct wl_listener *listener, void *data)
 	free(input);
 }
 
-static bool
-is_touch_device(struct wlr_input_device *wlr_input_device)
+static enum device_type
+wlr_device_to_device_type(struct wlr_input_device *wlr_input_device)
```

Comment:
We shouldn't use the `wlr_` prefix as that's 'reserved' for wlroots.
Suggest `device_type_from_wlr_device()` or something like that instead.

Reply:
Renamed to `device_type_from_wlr_device`